package/drbd.firewalld.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <service>
   <short>SUSE YaST DRBD</short>
-  <description>This allows you to open various ports related to SUSE YaST DRBD. Port 7788 and higher is recommend.</description>
+  <description>This allows you to open various ports related to SUSE YaST DRBD including Linstor support. Port 7788 and higher is recommend.</description>
```

Comment:
The description mentions only the port 7788, what about the newly added ports? Should we mention them as well?

Reply:
@lslezak 7788 is for the generic communication of DRBD. User need specific customized port for each DRBD (resource) connection, which is not a pre-defined ports but configurable. Mentioned 7788 here is to recomend user use ports >= 7788, not the same situation like this one.
This pr newly add new ports for DRBD linstor(like a warpper to manage DRBD in SDS/k8s environment). The ports is for services of Linstor, instead of connections.

It is not a must to add more description for each port, but i think your idea is good for users to understand the usage of the ports. I will modify the description in the later commit.